automation/src/test/scala/org/broadinstitute/dsde/firecloud/api/Orchestration.scala

```diff
@@ -132,7 +132,7 @@ trait Orchestration extends FireCloudClient with LazyLogging with SprayJsonSuppo
         aclEntries.map{ e => e.toMap })
     }
 
-    def setAttributes(namespace: String, name: String, attributes: Map[String, String])(implicit token: AuthToken): Unit = {
+    def setAttributes(namespace: String, name: String, attributes: Map[String, Any])(implicit token: AuthToken): Unit = {
```

Comment:
This `Any` could become an `Either[String, Seq[String]]`

Unfortunately, the changes are far reaching and add a lot of syntax:

```
val tags: Map[String, Either[String, Seq[String]]] = Map(
  "tag:tags" -> Right(Seq("testing","diabetes"))
)
```
and
```
val testAttributes = Map("A-key" -> Left("A value"), "B-key" -> Left("B value"), "C-key" -> Left("C value"))
```
So feel free to ignore this suggestion.

Reply:
i like the idea, but i'm using a pretty simple object to json mapper to send the data and my tags map gets serialized to this: {"tag:tags":{"r":["testing","diabetes"]}}

Reply:
Ah, hadn't come across that

Reply:
we'd really want this to be `Map[String, Attribute]` I think ... but that would require pulling in rawls-model, which is bigger than desired for this PR. How about just add a comment explaining what you expect the `Any` to be?